fitbenchmarking/fitbenchmarking
# 334 add cutest

#### Description of Work

Fixes #334

#### Testing Instructions

1. Run with the SIF file problems and check it works
2. Some of the SIF and NIST definitions are the same -- do these give the same results?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
